src/ucp/wireup/select.c

```diff
@@ -923,8 +923,9 @@ ucs_status_t ucp_wireup_select_lanes(ucp_ep_h ep, const ucp_ep_params_t *params,
             key->am_lane = lane;
         }
         if (lane_descs[lane].usage & UCP_WIREUP_LANE_USAGE_RNDV) {
-            ucs_assert(key->rndv_lane == UCP_NULL_LANE);
-            key->rndv_lane = lane;
+            /* TODO: add rndv sort */
+            ucs_assert(key->rndv_lanes[0] == UCP_NULL_LANE);
```

Comment:
How will you select what lanes are suitable for RNDV?

Reply:
don't know for a while, this is in nearest plans :)